lumigator/python/mzai/jobs/evaluator_lite/evaluator_lite/eval_lite.py

```diff
@@ -0,0 +1,95 @@
+import json
+from pathlib import Path
+
+import click
+import s3fs
+from datasets import load_from_disk
+from eval_metrics import EvaluationMetrics
+from evaluate_config import EvalJobConfig
+from loguru import logger
+
+
+def save_to_disk(local_path: Path, data_dict: dict):
+    logger.info(f"Storing evaluation results into {local_path}...")
+    local_path.parent.mkdir(exist_ok=True, parents=True)
+    with local_path.open("w") as f:
+        json.dump(data_dict, f)
+
+
+def save_to_s3(config: dict, local_path: Path, storage_path: str):
+    s3 = s3fs.S3FileSystem()
+    if storage_path.endswith("/"):
+        storage_path = "s3://" + str(
+            Path(storage_path[5:]) / config.get("name") / "eval_results.json"
+        )
+    logger.info(f"Storing evaluation results into {local_path}...")
+    s3.put_file(local_path, storage_path)
+
+
+def save_outputs(config: dict, eval_results: dict) -> Path:
+    storage_path = config.get("evaluation").get("storage_path")
+
+    local_path = Path("eval_results.json")
```

Comment:
```suggestion
    local_path = Path("results.json")
```

If we want to change the default local path where results are stored, it'd be useful to raise a flag (eg an issue) so we can make this consistent with the only other job, i.e. inference.

Reply:
Addressed with https://github.com/mozilla-ai/lumigator/pull/523/commits/1d81e87220820a3f2ee886f319810f992ef8ec49